docs/release-notes.md

```diff
@@ -194,6 +182,19 @@ This release adds the PGVector Vector Store node. Use this node to interact with
 
 </div>
 
+<div class="n8n-new-features" markdown>
+
+#### See active collaborators on workflows
+
+We added collaborator avatars back to the workflow canvas. You will see other users who are active on the workflow, preventing you from overriding each other's work.
+
+<figure>
```

Comment:
I have not yet added images to the docs either. This is the format that the [screenshots page in the docs repo wiki](https://github.com/n8n-io/n8n-docs/wiki/Screenshots#adding-to-docs) recommends:

```markdown
<figure markdown="span">
![Collaboration avatars](/_images/release-notes/Collaboration-avatar.png)
<figcaption>Collaboration avatars</figcaption>
</figure>
```

But I'm not sure that it makes too much of a difference since that should be rendered fairly similar.

Reply:
The thing I got from GPT is that `figcaption` is outdated and CSS should be used instead. 🤷 

I'll commit your version and see the change.

Reply:
Well that looks exactly the same :smile: I guess the center and italics are already part of the styling for that. Good to know!